Automattic/wp-calypso
# Add January Bump card to My Home screen

Related to #

## Proposed Changes

This adds a task card for the January Bump promo.

More info: pdvytD-Uc-p2#comment-696


## Testing Instructions




* Sandbox `public-api.wordpress.com`
* Apply  169188-ghe-Automattic/wpcom
* Go to the My Home screen for a pre and post blogs
* You should see the January Bump card

#### Post Launch
<img width="480" alt="Screenshot 2024-12-20 at 13 28 46" src="https://github.com/user-attachments/assets/f0fca6c0-f625-400d-8b49-400cc76014be" />

#### Pre Launch
* Ensure the pre-launch checklist is visible beneath the card

<img width="480" alt="Screenshot 2024-12-23 at 14 15 13" src="https://github.com/user-attachments/assets/32913894-d54d-46a3-b56c-dc409a4be140" />


## Pre-merge Checklist
